cinema/serializers.py

```diff
@@ -1 +1,72 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Actor, Genre, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.CharField(source="__str__", read_only=True)
```

Comment:
I've never seen such an implementation with "__str__" as source :0. Let's create a property instead because other devs won't be sure that __str__ indeed contains first and last name + it may be changed in the future, but property will be the same